build.gradle

```diff
@@ -38,6 +42,9 @@ buildscript {
 
 plugins {
   id 'nebula.dependency-recommender' version '9.0.2'
+  // Since 7.x gradle-docker-plugin is compiled using JDK11, thus using the latest version will fail Java8 builds
+  // https://bmuschko.github.io/gradle-docker-plugin/current/user-guide/#change_log
```

Comment:
This comment doesn't make it clear why this plugin is used. Can you explain in more detail?

Reply:
Ack.